swaggen/src/test/java/tests/yaml/YamlFileWriterTest.java

```diff
@@ -30,7 +30,7 @@
 	 * @throws JsonMappingException
 	 * @throws IOException
 	 */
-	@Test
+	//@Test
```

Comment:
Fair enough if we want to remove this test, but lets actually remove it then. We have 3 options here:
1. We want to refurbish the test later -> Add an @Ignore annotation with the @Test annotation.
2. We want to keep the code to be used by other tests -> Delete the @Test annotation and refactor this method to somewhere more appropriate.
3. We no longer need this code -> Delete the method entirely.
Don't leave commented out code.

Reply:
The test is meant to be there, my bad. It has been uncommented.